va/va.go

```diff
@@ -620,6 +620,15 @@ func (va *ValidationAuthorityImpl) doRemoteOperation(ctx context.Context, op rem
 			firstProb = currProb
 		}
 
+		// If enough perspectives have passed, start a timer for how long we'll wait for remaining perspectives
+		if len(passed) >= required && len(passedRIRs) >= requiredRIRs {
```

Comment:
This line is copied from below - should it be hoisted into a function?

Reply:
Having the same conditional in two places is fine imo; I'd be more concerned if this was the third.